apps/src/lib/client/tx.rs

```diff
@@ -94,8 +94,11 @@ async fn broadcast_tx(
     let mut client =
         TendermintWebsocketClient::open(WebSocketAddress::try_from(address)?)?;
     // It is better to subscribe to the transaction before it is broadcast
-    let query = Query::from(EventType::Tx)
-        .and_eq("tx.hash", hash_tx(&tx_bytes).to_string());
+    //
+    // Note that the `applied.hash` key comes from a custom event
+    // created by the shell
+    let query = Query::from(EventType::NewBlock)
+        .and_eq("applied.hash", hash_tx(&tx_bytes).to_string());
     client.subscribe(query)?;
     println!(
         "{:?}",
```

Comment:
```suggestion
        "Transaction added to mempool: {:?}",
```

Reply:
it's not letting me comment on that line here, but let's also maybe change line 112 to `"Transaction applied with result: {}",`